subo/util/exec.go

```diff
@@ -11,11 +11,20 @@ import (
 
 // Run runs a command, outputting to terminal and returning the full output and/or error.
 func Run(cmd string) (string, error) {
-	return RunInDir(cmd, "")
+	return run(cmd, "", false)
 }
 
 // RunInDir runs a command in the specified directory and returns the full output or error.
 func RunInDir(cmd, dir string) (string, error) {
+	return run(cmd, dir, false)
+}
+
+// RunSilent runs a command without printing to stdout and returns the full output or error.
+func RunSilent(cmd string) (string, error) {
```

Comment:
this isn't used anywhere at the moment. What are the plans for this function? 🙂

Reply:
I believe this is used in SCN?

Reply:
A search (case-insensitive) for the function name in atmo, scn, sat returned zero results 🤔